### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by Akasurde**
* Fix flake8 warning
* Minor spelling fixes

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

### CODE REVIEW

File: xmpp/browser.py [hunk 1]
```diff
@@ -61,65 +62,80 @@ class Browser(PlugIn):
...
+            cur = self._handlers['']
+        if node is None:
+            node = [None]
+        else:
+            node = node.replace('/', ' /').split('/')
         for i in node:
-            if i!='' and i in cur: cur=cur[i]
-            elif set and i!='': cur[i]={dict:cur,str:i}; cur=cur[i]
-            elif set or '' in cur: return cur,''
-            else: return None,None
```
Comment (snakeru) [inline xmpp/browser.py:118]: Just a random drive-by comment. 
Original xmpppy code was totally off the common styling rules, but it still had a certain style :)

Basically it was: the whole file must fit on my (vertically oriented) 19" screen. Therefore I tried to stay on the same line as much as possible.

Comment (snakeru) [reply xmpp/browser.py:118]: Actually, I'd recommend to not take this change if there are still any other forks out there that have valuable, but not-yet-merged changes. Whitespace changes might be very disrupting to carrying diffs around.

**Comment by amotl** [comment]
Dear Abhijeet,

thanks a stack for putting energy into this patch. I am sorry to reject its integration into the code base, following the advice from its original author.

With kind regards,
Andreas.
